torch_xla/csrc/tensor.h

```diff
@@ -26,6 +27,13 @@
 
 namespace torch_xla {
 
+class TORCH_API XLASymIntNodeImpl : public c10::SymIntNodeImpl {
+ public:
+  XLASymIntNodeImpl(torch::lazy::NodePtr ptr) : node_(std::move(ptr)){};
+  c10::SymIntNode mul(const c10::SymIntNode& other) override;
+  torch::lazy::NodePtr node_;
```

Comment:
random question, do we want `node_` to be private and provide a getter? I am OK if upstream also mark it as a public member.

Reply:
Made it private and provided a getter. That's indeed better.